### PR TITLE
Re-design new UI for 'my-requests' page with updated API

### PR DESCRIPTION
**Pull request by nimishawilson**
Redesign the UI for  'my-requests' page as shown in [prototype design](https://xd.adobe.com/view/7c6474dd-3491-4dc1-b5f9-4c91301be22e-2937/screen/1dc7e9b6-c6d8-4bfb-b132-fc560f8033a7/specs) and show updated API response.

## Tasks
- [x] Remove the heading 'my-requets'
- [x] Make UI changes for all the tab buttons and create new request button
- [x] Show the status of each request and update the request color accordingly
- [x] Update the API response when fetching requests
- [ ] Add 'requests' and 'settings' button at bottom
- [ ] Redirect to settings page on clicking the bottom navigation tab button.

## Screenshots
Pixel 2 XL
 

## Related Issue
[User Story QH-26](https://quarantinehelp.atlassian.net/browse/QH-26)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have removed things like [WIP] or other and reread the PR.
- [x] My code follows the code style of this project, passes TSLint checks.
- [x] I've done a rebase on `develop` and my feature branch is up to date.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by sebinbenjamin — CHANGES_REQUESTED** [review]
I think it would be good to have another look at the font sizes and text shadows.

**Comment by nimishawilson** [comment]
> I think it would be good to have another look at the font sizes and text shadows.

File: src/app/components/request/request.component.html [hunk 1]
```diff
@@ -9,16 +9,27 @@ <h1 class="ion-no-margin"> {{ requestData.createdAt | date:'dd' }} </h1>
           <span> {{ requestData.createdAt | date:'yyyy' }} </span>
         </div>
       </ion-col>
-      <ion-col size="10">
-        <ion-icon color="danger" *ngIf="requestData.type=='M'" name="medkit"> </ion-icon>
+      <ion-col class="request-content" size="10">
+        <ion-icon class="dollar-icon" *ngIf="requestData.hasIncentive" src="assets/common/dollar.svg"></ion-icon>
+        <ion-icon class="category-icon" *ngIf="requestData.type=='M'" name="medkit"> </ion-icon>
+        <ion-icon class="category-icon" *ngIf="requestData.type=='G'" name="cart"> </ion-icon>
+        <ion-icon class="category-icon" *ngIf="requestData.type=='O'" name="grid"></ion-icon>
```
Comment (sebinbenjamin) [inline src/app/components/request/request.component.html:16]: Try replacing with ng switch

File: src/app/pages/my-requests/my-requests.page.scss [hunk 2]
```diff
@@ -22,6 +22,25 @@ ion-icon {
...
+}
+
+ion-segment-button {
+  --background-checked: var(--ion-color-dark);
+  --color-checked: #ffffff;
+  --color: var(--ion-color-dark);
+  --border-color: var(--ion-color-dark);
+  --border-style: solid;
+  --border-width: 1px;
+}
```
Comment (sebinbenjamin) [inline src/app/pages/my-requests/my-requests.page.scss:46]: Please add a newline here

File: src/app/components/request/request.component.ts [hunk 3]
```diff
@@ -13,9 +15,22 @@ export class RequestComponent implements OnInit {
...
+
+  ngOnInit() {
+    console.log(this.requestData);
+
+    // TODO: Find out about the other statuses - C & F
+    this.requestStatusColor = 'primaryAF';
+    this.requestStatusSecondaryColor = 'secondaryAF';
+    if (this.requestData.status === RequestStatus.transit) {
+      this.requestStatusColor = 'primaryHL';
+      this.requestStatusSecondaryColor = 'secondaryHL';
```
Comment (sebinbenjamin) [inline src/app/components/request/request.component.ts:31]: Aren't the colours based on the userType?

**Review by sebinbenjamin — CHANGES_REQUESTED** [review]

File: src/app/components/request/request.component.ts (hunk 3 above)
Comment (nimishawilson) [reply src/app/components/request/request.component.ts:31]: @sebinbenjamin No. These colors are based on request status.

File: src/app/pages/my-requests/my-requests.page.scss (hunk 2 above)
Comment (nimishawilson) [reply src/app/pages/my-requests/my-requests.page.scss:46]: added it

File: src/app/components/request/request.component.html (hunk 1 above)
Comment (nimishawilson) [reply src/app/components/request/request.component.html]: Replaced with ng-switch.

**Review by sebinbenjamin — APPROVED** [review]

**Pull request merged**